ntbs-integration-tests/NotificationPages/ClinicalDetailsPageTests.cs

```diff
@@ -97,6 +97,31 @@ public async Task PostDraft_ReturnsPageWithAllModelErrors_IfModelNotValid()
             resultDocument.AssertErrorSummaryMessage("ClinicalDetails-Notes", "notes", "Too many characters entered into the Notes field");
         }
 
+        [Fact]
+        public async Task PostDraft_ReturnsModelError_IfPostMortemAndStartedTreatmentBothSelected()
+        {
+            // Arrange
+            var url = GetCurrentPathForId(Utilities.DRAFT_ID);
+            var initialDocument = await GetDocumentForUrlAsync(url);
+
+            var formData = new Dictionary<string, string>
+            {
+                ["NotificationId"] = Utilities.DRAFT_ID.ToString(),
+                ["NotificationSiteMap[PULMONARY]"] = "true",
+                ["ClinicalDetails.StartedTreatment"] = "true",
+                ["ClinicalDetails.IsPostMortem"] = "true"
+            };
+
+            // Act
+            var result = await Client.SendPostFormWithData(initialDocument, formData, url);
+
+            // Assert
+            var resultDocument = await GetDocumentAsync(result);
+
+            result.EnsureSuccessStatusCode();
+            resultDocument.AssertErrorSummaryMessage("ClinicalDetails-IsPostMortem", null, "Unable to save value of ‘Diagnosis after death’ field due to conflicting information on the record: Started treatment is set to true");

```

Comment:
There are competing philosophies here: I would refer to the constant for the expected string, otherwise changing the value of the constant will cause the test to fail. (Some would say that that's exactly what you want to happen, but in practice the risk of someone accidentally changing the string is so low that I don't think it's worth it).